pyvista/themes.py

```diff
@@ -2058,6 +2060,26 @@ def save(self, filename):
         with open(filename, 'w') as f:
             json.dump(self.to_dict(), f)
 
+    @property
+    def use_ipyvtk(self):  # pragma: no cover
+        """Set or return the usage of "ipyvtk" as a jupyter backend.
+
+        Deprecated in favor of ``jupyter_backend``.
+        """
+        warnings.warn('use_ipyvtk is deprecated.  Please use '
+                      '``pyvista.global_theme.jupyter_backend =``', DeprecationWarning)
+        return self._use_ipyvtk
```

Comment:
Sorry, I didn't catch this on first read. It seems that if `jupyter_backend` is updated, `use_ipyvtk` will be stale. Shouldn't we just return `self.jupyter_backend == 'ipyvtklink'` in the getter, and remove the `_use_ipyvtk` attr altogether?

Reply:
Good thinking.  Implemented in 9ab4c9108d1278afa651b5d2a94c80c22d50d17d